docs/dpdk/mlnx-dpdk/crd-dpdk-connectx4.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: "k8s.cni.cncf.io/v1"
+kind: NetworkAttachmentDefinition
+metadata:
+  name: sriov-dpdk-mlnx-connectx4
+  annotations:
+    k8s.v1.cni.cncf.io/resourceName: intel.com/mlnx_connectx4
+spec:
+  config: '{
+    "type": "sriov",
+    "name": "sriov-mlnx_connectx4",
+    "vlan": 2000
```

Comment:
@Mmduh-483 , do we really need to specify vlan here ?

Reply:
no